app/firebase/functions/src/posts/posts.repository.ts

```diff
@@ -121,11 +121,12 @@ export class PostsRepository extends BaseRepository<AppPost, AppPostCreate> {
     const posts = await paginated
       .limit(queryParams.fetchParams.expectedAmount)
       .get();
-
-    return posts.docs.map((doc) => ({
+    const appPosts = posts.docs.map((doc) => ({
       id: doc.id,
       ...doc.data(),
     })) as AppPost[];
+
+    return appPosts.sort((a, b) => b.createdAtMs - a.createdAtMs);
```

Comment:
This should not be need, the `orderBy` above should already return the ordered posts. Isn't it?